tools/code_analyzer/op_dependency.cpp

```diff
@@ -710,11 +768,14 @@ class OpDependency : public ModulePass {
     }
   }
 
-  static std::shared_ptr<std::string> extractOpSchema(Value* V) {
+  static std::shared_ptr<std::string> extractOpSchema(
+      const std::string& contextualNamespace, Value* V) {
     std::vector<std::string> schemaStrs;
     extractStringValue(V, [&](const std::string& str) {
-      if (FunctionSchemaPatternLoc.pattern->match(str)) {
-        schemaStrs.push_back(str);
+      const std::string& schemaStr =
```

Comment:
I love the smell of const reference extension in the morning

Reply:
yeah, it's intentional to avoid copying in no-namespace-case - is it bad? :>